### PR TITLE
bump roles jenkins, chrony

### PR DESCRIPTION
**Pull request by mira-miracoli**
part of:
- https://github.com/usegalaxy-eu/issues/issues/492

REQUIRES:
- https://github.com/usegalaxy-eu/ansible-chrony/pull/1

### CODE REVIEW

File: requirements.yaml [hunk 1]
```diff
@@ -61,12 +61,12 @@ roles:
...
   - name: geerlingguy.jenkins
-    version: 4.3.0
+    version: 5.1.0
   - name: geerlingguy.repo-epel
     version: 3.1.0
   - name: influxdata.chrony
-    src: https://github.com/usegalaxy-eu/ansible-chrony
-    version: 0.1.0
+    src: https://github.com/mira-miracoli/ansible-chrony
+    version: patch-1
```
Comment (bgruening) [inline requirements.yaml:69]: Is that temporary?

Comment (mira-miracoli) [reply requirements.yaml:69]: it is just fixed

**Review by bgruening — APPROVED** [review]

**Review by sanjaysrikakulam — APPROVED** [review]

File: group_vars/build.yml [hunk 2]
```diff
@@ -1,4 +1,29 @@
...
+  - curl
+  - tmux
+  - git
+  - nmap
+  - tcpdump
+  - net-tools
+  - unzip
+  - tmpwatch
+  - rclone
+  - "{{ byobu if ansible_distribution_major_version < '9' | default(omit) }}"
```
Comment (kysrpex) [inline group_vars/build.yml:20]:
```suggestion
  - "{{ 'byobu' if ansible_distribution_major_version < '9' else omit }}"
```

Same story as [here](https://github.com/usegalaxy-eu/infrastructure-playbook/pull/998#discussion_r1397117191).

**Review by kysrpex — CHANGES_REQUESTED** [review]
Also you included [d3f3e3d](https://github.com/usegalaxy-eu/infrastructure-playbook/pull/999/commits/d3f3e3d60aefabfd3971452d21d640f8312786a4) and [92b634b](https://github.com/usegalaxy-eu/infrastructure-playbook/pull/999/commits/92b634b2acdf4b6e65b4cb79cfd664912a42a685) from https://github.com/usegalaxy-eu/infrastructure-playbook/pull/998. I guess this was not meant to be the case.

**Review by sanjaysrikakulam — APPROVED** [review]

File: group_vars/build.yml (hunk 2 above)
Comment (kysrpex) [reply group_vars/build.yml:20]: Not resolved :(

**Review by kysrpex — APPROVED** [review]

**Pull request merged**